srv/salt/state/conda_env_with_dask.sls

```diff
@@ -0,0 +1,33 @@
+# File:/srv/salt/conda_env_with_dask.sls
+
+check_conda_env_py39_dask:
+  cmd.run:
+    - name: 'conda env list | grep "py39_dask"'
+    - shell: /bin/bash
+    - env:
+      - PATH: /root/miniconda3/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin
+    - require:
+      - check_conda_install  # miniconda.sls; will ensure conda is set up.
+
+create_conda_env_py39_dask:
+  cmd.run:
+    - name: 'conda create --name py39_dask python=3.9'
+    - shell: /bin/bash
+    - env:
+      - PATH: /root/miniconda3/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin
+    - onfail:
+      - cmd: check_conda_env_py39_dask
+
+activate_conda_env_py39_dask:
+  cmd.run:
+    - name: 'source ~/miniconda3/etc/profile.d/conda.sh && conda activate py39_dask'
+
+check_dask_install_in_ev_py39_dask:
+  cmd.run:
+    - name: 'source ~/miniconda3/etc/profile.d/conda.sh && conda activate py39_dask && conda list | grep "dask"'
+
```

Comment:
This whole bit shouldnt be necessary unless some how conda force installs dask even if already present in its environment.

Reply:
That's true, and I might have had that as an attempted workaround that didn't address a problem I was having.
It might also have been way of [this comment](https://github.com/saltstack/salt/issues/52595#issuecomment-485509687) were  this state pattern to be used for many packages.

Reply:
The activate env isn't necessary as it does nothing but activate the env. I'm not sure if I thought the activation persisted across states, or if it was an initial check that the env existed, but it serves no purpose.